lib/tasks/whitehall_importer.rb

```diff
@@ -180,10 +180,7 @@ def create_edition(document, translation, whitehall_edition, edition_number)
         number: edition_number,
         revision_synced: true,
         revision: revision,
-        status: Status.new(
-          state: state(whitehall_edition),
-          revision_at_creation: revision,
-        ),
+        status: status(whitehall_edition, revision),
```

Comment:
Probably worth liaising with @leenagupte on this change, since she's introduced a method here with a different name: https://github.com/alphagov/content-publisher/pull/1438/files#diff-713c586411d3e8a35468b374fe29c4a8R135